codegen/src/test/java/org/apache/servicecomb/toolkit/codegen/GeneratorTest.java

```diff
@@ -86,5 +90,13 @@ public void testGetCodeGeneratorInstanse() {
     CodeGenerator unknownCodeGenerator = GeneratorFactory.getGenerator(CodeGenerator.class, "unknown");
     Assert.assertNull(unknownCodeGenerator);
   }
+
+  @Test
+  public void externalConfigGenerator() {
```

Comment:
I don't think we need to add unit test for constant.

Reply:
Thank you for reminding me, I will modify it in the next pr